engine/cmd_config.go

```diff
@@ -203,6 +204,14 @@ var configCheckCmd = &cobra.Command{
 			}
 		}
 
+		if conf.CDN != nil && conf.CDN.API.HTTP.URL != "" {
```

Comment:
for the moment i think conf.CDN.API.HTTP.URL is useless

Reply:
for the moment, all changes in this file is useless in fact, but we will need it for the next step.